js/index.js

```diff
@@ -64,4 +66,48 @@ const messageList = messageSection.querySelector('ul');
     messageList.appendChild(newMessage);
     messageSection.hidden = false;
     messageForm.item(0).reset();
-});
\ No newline at end of file
+});
+
+const githubRequest = new XMLHttpRequest();
+githubRequest.open('GET', 'https://api.github.com/users/raissanfg/repos');
+githubRequest.send();
+githubRequest.addEventListener('load', function() {
+const repositories = JSON.parse(this.response)
+
+const projectSection = document.querySelector('#projects')
+const projectList = projectSection.querySelector('ul')
+
+ for (let i=0; i < repositories.length; i++) {
+    var project = document.createElement('li');
+    var repositoryLink = document.createElement('a');
+    repositoryLink.href = repositories[i].html_url;
+    repositoryLink.innerHTML = repositories[i].name;
+    projectList.appendChild(project);
+    project.appendChild(repositoryLink);
+
+ }
+});
+
+function renderProjectsWithFetch() {
+    fetch('http://api.github.com/user/raissanfg/repos')
+    .then((res) => res.json())
+    .then((data) => {
+
+    const repositories = JSON.parse(this.response)
+    const projectSection = document.querySelector('#projects')
+    const projectList = projectSection.querySelector('ul')
+
+         for (let i=0; i < repositories.length; i++) {
+         var project = document.createElement('li' );
+         var repositoryLink = document.createElement('a');
+         repositoryLink.href = repositories[i].html_url;
+         repositoryLink.innerHTML = repositories[i].name;
+         projectList.appendChild(project);
+         project.appendChild(repositoryLink);
+         proja.setAttribute('id', 'proja')
+        
+ }
+    })
+    renderProjectsWithFetch();
```

Comment:
This function call `renderProjectsWithFetch();`, is right now _inside_ the brackets for the function renderProjectsWithFetch. That means the Javascript  will never actually reach this line to call the code. This should be moved down to below Line 112.